cli/src/main.rs

```diff
@@ -841,3 +984,48 @@ fn get_pair_from_str(account: &str) -> sr25519::AppPair {
 		},
 	}
 }
+
+pub fn count_exchange_rate_updated_events<P: Pair, Client: 'static>(
+	api: Api<P, Client>,
+	duration: Duration,
+) -> i32
```

Comment:
Not unsigned?

Reply:
If you think `i32` is the correct value, that's fine with me. But could you explain?

Reply:
No it should be unsigned.